plugins/vpp/wireguardplugin/vppcalls/vpp2009/vppcalls_handlers.go

```diff
@@ -0,0 +1,84 @@
+//  Copyright (c) 2020 Doc.ai and/or its affiliates.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at:
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package vpp2009
+
+import (
+	"fmt"
+	govppapi "git.fd.io/govpp.git/api"
+	"go.ligato.io/cn-infra/v2/logging"
+	"net"
+	"strconv"
+	"strings"
+
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2009"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2009/ip_types"
+	vpp_wg "go.ligato.io/vpp-agent/v3/plugins/vpp/binapi/vpp2009/wireguard"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/ifplugin/ifaceidx"
+	"go.ligato.io/vpp-agent/v3/plugins/vpp/wireguardplugin/vppcalls"
+)
+
+func init() {
+	var msgs []govppapi.Message
+	msgs = append(msgs, vpp_wg.AllMessages()...)
+
+	vppcalls.AddHandlerVersion(vpp2009.Version, msgs, NewWgVppHandler)
+}
+
+type WgVppHandler struct {
+	callsChannel govppapi.Channel
+	ifIndexes    ifaceidx.IfaceMetadataIndex
+	log          logging.Logger
+}
+
+func NewWgVppHandler(ch govppapi.Channel, ifIdx ifaceidx.IfaceMetadataIndex, log logging.Logger) vppcalls.WgVppAPI {
+	return &WgVppHandler{ch, ifIdx, log}
+}
+
+func IPToAddress(ipstr string) (addr ip_types.Address, err error) {
+	netIP := net.ParseIP(ipstr)
+	if netIP == nil {
+		return ip_types.Address{}, fmt.Errorf("invalid IP: %q", ipstr)
+	}
+	if ip4 := netIP.To4(); ip4 == nil {
+		addr.Af = ip_types.ADDRESS_IP6
+		var ip6addr ip_types.IP6Address
+		copy(ip6addr[:], netIP.To16())
+		addr.Un.SetIP6(ip6addr)
+	} else {
+		addr.Af = ip_types.ADDRESS_IP4
+		var ip4addr ip_types.IP4Address
+		copy(ip4addr[:], ip4)
+		addr.Un.SetIP4(ip4addr)
+	}
+	return
+}
+
+func IPToPrefix(prefstr string) (prefix ip_types.Prefix, err error) {
```

Comment:
Same here, use the generated conversion func:

https://github.com/ligato/vpp-agent/blob/master/plugins/vpp/binapi/vpp2009/ip_types/ip_types.ba.go#L509

Reply:
Done